tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -1170,6 +1196,10 @@ namespace tuplex {
                     rowDelta += taskNonConformingRows.size();
                 }
 
+                if (tstage->hasOutputLimit()) {
+                    trimPartitionsToLimit(output, tstage->outputTopLimit(), tstage->outputBottomLimit());
```

Comment:
add comment what kind of partitions this func would expect (i.e., how do they need to look like for bottom limit??)

Reply:
done!